sklearn/tests/test_pipeline.py

```diff
@@ -48,6 +48,12 @@ class T(IncorrectT):
     def fit(self, X, y):
         return self
 
+    def get_params(self, deep=False):
+        return {'a': self.a, 'b': self.b}
+
+    def set_params(self, **params):
+        self.a = params['a']
```

Comment:
The test ```test_pipeline_init``` only tested setting one of the parameters via ```set_params```. I can add one for ```b``` as well in case it gets tested somewhere in the future.